Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/ViewBookmarkLoader.cpp

```diff
@@ -354,32 +403,55 @@ namespace AzToolsFramework
                 {
                     // TODO: check that the file actually exists
                     m_bookmarkfileName = bookmarkComponent->GetLocalBookmarksFileName();
+                    LoadViewBookmarks();
+
+                    for (int i = 0; i < m_localBookmarks.size(); i++)
```

Comment:
suggestion: A comment to explain this would be good (again indicating this is a temporary solution until a better UX alternative is created).